### PR TITLE
refactor(application-table): use new table for application page

### PR DESCRIPTION
**Pull request by zverbeta**
Use a new table for the application page.

Switch between the old and new tables using configuration settings

### CODE REVIEW

**Review by oscaryald — APPROVED** [review]

**Review by andrewkikot — APPROVED** [review]

File: packages/entity/src/entity-list-card/entity-table/entity-table.component.ts [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                set(column, 'config', {
+                    routerLink,
+                    noDeepLink,
+                });
+            } else if (field.field === 'stateKey') {
+                set(column, 'selector', '@xm-ngx/components/application-table-state');
+            } else if (field.field === 'startDate' || field.field === 'updateDate') {
+                set(column, 'selector', '@xm-ngx/components/date');
+                set(column, 'config', {
+                    format: 'MM/dd/yy hh:mm',
```
Comment (sergeysenja1992) [inline packages/entity/src/entity-list-card/entity-table/entity-table.component.ts:155]: pls move date format to config

Comment (sergeysenja1992) [reply packages/entity/src/entity-list-card/entity-table/entity-table.component.ts:155]: pls make it configurable like hideDeleteButton

**Review by sergeysenja1992 — APPROVED** [review]

File: packages/entity/registry/index.ts [hunk 2]
```diff
@@ -5,4 +5,24 @@ export const XM_ENTITY_ELEMENTS: XmDynamicEntries = [
         selector: '@xm-ngx/components/navbar-search-widget',
```
Comment (Kaskyi) [inline packages/entity/registry/index.ts:5]: `@xm-ngx/components/navbar-search-widget` is a backward compatibility.
 `@xm-ngx/entity/...` could be better

Comment (zverbeta) [reply packages/entity/registry/index.ts:5]: I don't know about the widget, so i leave as is

**Pull request merged**